prediction/src/algorithms/identify/prediction.py

```diff
@@ -160,15 +199,13 @@ def predict_cubes(model_path, patient_id, magnification=1, ext_name=""):  # noqa
     if not os.path.exists(dst_dir):
         os.makedirs(dst_dir)
 
-    holdout_ext = ""
-    flip_ext = ""
-
-    dst_dir += "predictions" + str(int(magnification * 10)) + holdout_ext + flip_ext + "_" + ext_name + "/"
+    dst_dir += "predictions" + str(int(magnification * 10)) + "_" + ext_name + "/"
     if not os.path.exists(dst_dir):
         os.makedirs(dst_dir)
 
     model = get_net(input_shape=(CUBE_SIZE, CUBE_SIZE, CUBE_SIZE, 1),
                     load_weight_path=model_path)
+
```

Comment:
Please make these changes in separate commits next time - very useful for speed up review!

Reply:
Ok, I'll try to get rid of this habit. Thank you for the review!